app/assets/stylesheets/elements/inheritance.scss

```diff
@@ -0,0 +1,23 @@
+%city-objects {
```

Comment:
I'm not familiar with this selector - did you want a `.` here to indicate a css class selection?

Reply:
neat! today I learned about placeholder selectors: http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_foo

 I had always used the `@mixin` keyword